libnopegl/src/node_text.c

```diff
@@ -72,6 +72,11 @@ struct pipeline_desc_fg {
     int32_t user_transform_index;
     int32_t color_index;
     int32_t opacity_index;
+    int32_t outline_index;
+    int32_t outline_color_index;
+    int32_t glow_index;
+    int32_t glow_color_index;
+    int32_t blur_index;
```

Comment:
You will need to pack data and use less attributes: and make that work on macOS/openGL:
```
[ERROR] src/backends/gl/program_gl.c:71 program_check_status: could not link shader: ERROR: Implementation limit of 16 MAX_VERTEX_ATTRIBS (e.g., number of generic plus conventional active vec4 attributes) exceeded, shader uses up to vec4 attribute 16.
```
See https://github.com/nope-project/nope.gl/pull/103.